.gitignore

```diff
@@ -40,3 +40,4 @@ yarn-debug.log*
 .yarn-integrity
 config/database.yml
 .env
+.rubocop.yml
```

Comment:
I think you need to track this file on `git`! 
remove it and add the `.rubocop.yml`

Reply:
okay, we work in it.

Reply:
Ready, we solve it.